src/components/ApproveRequests.vue

```diff
@@ -50,7 +59,13 @@ export default {
       this.loading = true;
       const campaignInstance = await Campaign.at(address)
       console.log(campaignInstance)
-      const result = await campaignInstance.approveRequest(ethId, { from: this.$store.state.accountNum})
+      let result;
+      try {
+        result = await campaignInstance.approveRequest(ethId, { from: this.$store.state.accountNum})
+      } catch(error) {
+        console.log(error)
+        this.loading = false;
+      }
```

Comment:
Since try/ catch locally scopes variables, I had to initialize result outside of them and assign result inside for use later in the function